plugins/fix-china-map-offset.js

```diff
@@ -1,10 +1,12 @@
 ﻿// @author         modos189
 // @name           Fix maps offsets in China
 // @category       Tweaks
-// @version        0.3.0
+// @version        0.3.1
 // @description    Show correct maps for China user by applying offset tweaks.
 
 
+/* exported setup --eslint */
+/* global L */
```

Comment:
This should be defined globally.

Reply:
This changes from file to file, so having it in the same place without the need to look up at different places is more convenient.